src/hooks/useFocusElement.ts

```diff
@@ -0,0 +1,24 @@
+import { useRef, RefObject } from 'react';
+
+/**
+ * A hook that focus a referenced element.
+ *
+ * @param {FocusOptions} options
+ * @returns {[() => void, RefObject<T>]}
+ */
+export const useScrollToElement = <T extends HTMLElement>(
```

Comment:
Is the function used to scroll to Element to focus an element? 

nit: focuses*

Reply:
My bad. This needs renaming